File: Python/find_minimum_value.py
# p71
def find_minimum_value(datas: list) -> int:
    min = 9999999

    for data in datas:
        if min > data:
            min = data

    return min


datas = [540, -238, 190, 22, -645, 832, 777, -542, 0, 395]
result = find_minimum_value(datas)

print("最小値 = %d" % result)

print("最小値 = %d" % min(datas))

File: TypeScript/FindMinimumValue.ts
// p71
const FindMinimumValue = (datas: Array<number>) => {
    let min = 9999999

    for (const data of datas) {
        if (min > data) {
            min = data
        }
    }
    return min
}

const datas = [540, -238, 190, 22, -645, 832, 777, -542, 0, 395]
const result = FindMinimumValue(datas)

console.log(`最小値 = ${result}`)

console.log(`最小値 = ${Math.min(...datas)}`)